st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -382,7 +384,8 @@
             'password': {
```

Comment:
I guess the evolution step would be to `detect` some keywords as secret params by default? Like password, key, etc...